Libraries/Geolocation/RCTLocationObserver.m

```diff
@@ -234,6 +233,7 @@ - (void)timeout:(NSTimer *)timer
                                                         selector:@selector(timeout:)
                                                         userInfo:request
                                                          repeats:NO];
+  _pendingRequests = [NSMutableArray new];
```

Comment:
It is the only thing that adds to it, but what if you call getCurrentPosition() again before the first request returns?